backend/api/models.py

```diff
@@ -169,4 +169,19 @@ class Meta:
         constraints = [
             models.UniqueConstraint(fields=['swim_meet', 'num_event'], name='unique_num_event_swim_meet'),
             models.UniqueConstraint(fields=['swim_meet', 'event_type', 'group'], name='unique_group_event_type_swim_meet')
-        ]
\ No newline at end of file
+        ]
+        
+class Heat(models.Model):
+    event = models.ForeignKey(MeetEvent, on_delete=models.CASCADE, related_name='heat_event')
+    athlete = models.ForeignKey(Athlete, on_delete=models.SET_NULL, blank=True, null=True, related_name='heat_athlete')
+    lane_num = models.PositiveSmallIntegerField()
+    seed_time = models.CharField(max_length=15, null=True, blank=True, help_text='Time in minutes:seconds.milliseconds format')
+    heat_time = models.CharField(max_length=15, null=True, blank=True, help_text='Time in minutes:seconds.milliseconds format')
```

Comment:
With the field time change introduced by #72, we will need to update the field type for `seed_time` and `heat_time` to DurationField too.

Reply:
I made the changes.  I added blank=True and null=True, because seed time and heat time could be null.